sormas-ui/src/main/java/de/symeda/sormas/ui/visit/VisitGrid.java

```diff
@@ -64,19 +65,21 @@ public VisitGrid(VisitCriteria criteria) {
 			VisitIndexDto.VISIT_STATUS,
 			VisitIndexDto.VISIT_REMARKS,
 			VisitIndexDto.DISEASE,
-			VisitIndexDto.SYMPTOMATIC,
```

Comment:
Replace all references to SymptomsDto that you introduced with VisitIndexDto again; only the I18N_PREFIX should be taken from SymptomsDto (because symptomatic and temperature are still properties of VisitIndexDto here, we're only interested in changing where the caption is generated from)